test/ExportVerilog/rtl-dialect.mlir

```diff
@@ -7,7 +7,7 @@ module {
 
```

Comment:
Doesn't this patch break the arrays in rtl-dialect.mlir:@wires that are checking for printing as unpacked syntax?

Reply:
I kept inout arrays as unpacked arrays. There are several operations which only interact with them I'm confused as to what your intent was with them. ArrayIndex in particular only works with InOut arrays, but is named to imply that it works on arrays in general.

I can switch them to packed if they're just meant to model bi-directional wire arrays, as I would have imagined.

Reply:
I think rtl.array should always mean packed arrays for self consistency.

My only interest in unpacked arrays that that is what Chisel currently generates.  I don't understand if that is important for some reason, but I'd rather retain compatibility with its current output rather than changing it right now.  I can introduce a new sv type for that.